### DISCUSSION TITLE
Vercel API verify project domain 403 error

### DISCUSSION THREAD

**Discussion by hmjk99** [vercel/community#4672] [Help] [answered]
### Summary

Hello. I am using the vercel api to add, delete, and more for domains in my application using React. I can successfully, delete, add, check configuration for the domain using the Vercel Api. But for some reason, I cannot check the verification. It is giving me a 403 error. I am using the same token for all the requests. 

### Steps to Reproduce

<img width="809" alt="Screenshot 2023-10-30 at 7 29 40 PM" src="https://github.com/vercel/community/assets/124408498/459a52a8-506a-4787-988f-a86f28ee83c1">
<img width="742" alt="Screenshot 2023-10-30 at 7 30 57 PM" src="https://github.com/vercel/community/assets/124408498/dcd1ab1f-f341-49a0-97cf-77333db23a9d">

**Comment by amyegan** [comment, marked as answer]
Please double check that the `apiToken` variable is set. Missing or incorrect tokens are a common cause of [403 Forbidden](https://vercel.com/docs/rest-api/errors#forbidden) errors.
